shared/ui/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,49 @@
+import { ComponentPropsWithoutRef } from 'react';
+
+import { Typography } from '@/shared/ui';
+import * as Radio from '@radix-ui/react-radio-group';
+import { clsx } from 'clsx';
+
+import style from './radio-group.module.scss';
+
+export type OptionsRadioGroup = {
+  disabled?: boolean;
+  id?: string;
+  label?: string;
+  value: string;
+}[];
```

Comment:
I think it is better to define the `OptionRadioGroup` type without square brackets, that is, as an array element rather than an array.

This type is assumed to be imported by any other code. In such case you always should extract items type.
###### Now:
```typescript
type ExtendedComponentProps = {
  option: OptionsRadioGroup[number];
}
```
##### Better:
```typescript
type ExtendedComponentProps = {
  option: OptionsRadioGroup;
}
```